kubectl-minio/cmd/tenant-info.go

```diff
@@ -54,53 +53,63 @@ func newTenantInfoCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 		Short: "List all volumes in existing tenant",
 		Long:  infoDesc,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if err := c.validate(); err != nil {
+			if err := c.validate(args); err != nil {
 				return err
 			}
-			return c.run()
+			return c.run(args)
 		},
 	}
-
+	cmd = helpers.DisableHelp(cmd)
 	f := cmd.Flags()
-	f.StringVar(&c.name, "name", "", "name of the MinIO tenant to list volumes")
 	f.StringVarP(&c.ns, "namespace", "n", helpers.DefaultNamespace, "namespace scope for this request")
 	return cmd
 }
 
-func (d *infoCmd) validate() error {
-	if d.name == "" {
-		return errors.New("--name flag is required for adding volumes to tenant")
+func (d *infoCmd) validate(args []string) error {
+	if args == nil {
+		return errors.New("provide the name of the tenant, e.g. 'kubectl minio tenant info tenant1'")
+	}
+	if len(args) != 1 {
+		return errors.New("info command supports a single argument, e.g. 'kubectl minio tenant info tenant1'")
+	}
+	if args[0] == "" {
+		return errors.New("provide the name of the tenant, e.g. 'kubectl minio tenant info tenant1'")
 	}
 	return nil
 }
 
 // run initializes local config and installs MinIO Operator to Kubernetes cluster.
-func (d *infoCmd) run() error {
+func (d *infoCmd) run(args []string) error {
 	// Create operator client
 	oclient, err := helpers.GetKubeOperatorClient()
 	if err != nil {
 		return err
 	}
-	err = listPoolsTenant(oclient, d)
+
+	tenant, err := oclient.MinioV1().Tenants(d.ns).Get(context.Background(), args[0], metav1.GetOptions{})
 	if err != nil {
 		return err
 	}
+	printTenantInfo(*tenant)
+
 	return nil
 }
 
-func listPoolsTenant(client *operatorv1.Clientset, d *infoCmd) error {
-	tenant, err := client.MinioV1().Tenants(d.ns).Get(context.Background(), d.name, metav1.GetOptions{})
-	if err != nil {
-		return err
+func printTenantInfo(tenant miniov1.Tenant) {
```

Comment:
Thought: would it be difficult to include in the output the connectivity information? e.g. the service endpoint for Tenant + Console? The credentials wont be there, but this seems like a nice way of helping users avoid doing `kubectl get svc` and figuring out what they need to use.

Reply:
makes sense. added